contracts/Cross-chain/OmnichainProposalSender.sol

```diff
@@ -115,22 +119,25 @@ contract OmnichainProposalSender is ReentrancyGuard, BaseOmnichainControllerSrc
      * @notice Sends a message to execute a remote proposal
      * @dev Stores the hash of the execution parameters if sending fails (e.g., due to insufficient fees)
      * @param remoteChainId_ The LayerZero id of the remote chain
-     * @param payload_ The payload to be sent to the remote chain. It's computed as follows: payload = abi.encode(targets, values, signatures, calldatas, proposalType)
+     * @param payload_ The payload to be sent to the remote chain.
+     * It's computed as follows: payload = abi.encode(targets, values, signatures, calldatas, proposalType)
      * @param adapterParams_ The params used to specify the custom amount of gas required for the execution on the destination
-     * @custom:event Emits ExecuteRemoteProposal with remote chain id and payload on success
+     * @param zroPaymentAddress_ The address of the ZRO token holder who would pay for the transaction.
```

Comment:
Just a small tip: let's not use any full stops in netspec since we're not using them anywhere.

Reply:
https://github.com/VenusProtocol/governance-contracts/pull/64/commits/422c2fdcc2b34357c532857bac087cdf7c09ff0f